src/components/molecules/pokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react';
+import Axios from 'axios';
+
+import { Card, Header, ImageContainer } from '../../atoms';
+import { Ability } from '../ability/Ability';
+
+import './PokemonCard.scss';
+
+export const PokemonCard = ({ url, name }) => {
+	const [avatar, setAvatar] = useState(null);
+	const [moves, setMoves] = useState(0);
+	const [weight, setWeight] = useState(0);
+	const [abilities, setAbilities] = useState(0);
+	const [isLoading, setIsLoading] = useState(false);
+
+	const cancelToken = Axios.CancelToken.source();
+
+	useEffect(() => {
+		const fetchCardData = async () => {
+			try {
+				setIsLoading(true);
```

Comment:
Je kunt de `isloading` het beste vóór het try/catch blok op `true` zetten en dan ná het blok weer op `false`, omdat we zowel bij een error als een succes uiteindelijk het try/catch blok weer verlaten